pyaedt/generic/DataHandlers.py

```diff
@@ -101,6 +101,16 @@ def _dict2arg(d, arg_out):
                 arg = ["NAME:" + k]
                 _dict2arg(el, arg)
                 arg_out.append(arg)
+        elif isinstance(v, list) and len(v) > 0 and all(isinstance(n, int) for n in v):
```

Comment:
This modification introduces a regression problem with other methods (mainly in Icepak). 
I suggest undoing these modifications and directly using it in both methods (get and set core loss), maybe you could create a private method used by these 2 methods.